pom.xml

```diff
@@ -7,18 +7,29 @@
     <groupId>org.example</groupId>
     <artifactId>practice-software-testing-hillel</artifactId>
     <version>1.0-SNAPSHOT</version>
+    <packaging>pom</packaging>
+    <modules>
+        <module>ui-tests</module>
+        <module>api-tests</module>
+    </modules>
 
     <properties>
         <maven.compiler.source>21</maven.compiler.source>
         <maven.compiler.target>21</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <restAssured.version>5.4.0</restAssured.version>
-        <junit.version>5.9.1</junit.version>
+        <junit.version>5.8.1</junit.version>
         <lombok.version>1.18.30</lombok.version>
         <jackson.version>2.17.1</jackson.version>
         <assertj.version>3.26.0</assertj.version>
+        <selenide.version>7.3.3</selenide.version>
     </properties>
 <dependencies>
```

Comment:
dependencies are not needed in the parent pom. only dependencyManagement section